WordPress/wporg-parent-2021
# Tools: Update the build script to build JS files too

⚠️ ⚠️

I started this thinking I needed to write more JS for #28, but when I realized it was just a CSS fix I stopped working on it. Pretty sure this is ready to go, but I don't want anyone to bother reviewing it right now.

---------

This copies over the build script from wporg-mu-plugins, with some tweaks since it only needs to run the single project, and needs to build sass. The JS file is now moved to `src`, and uses modern JS. This lays out the groundwork for adding more complex code to this file.

### How to test the changes in this Pull Request:

1. Run `yarn` to pull in new deps
2. Run `yarn workspace wporg-parent-2021 build`
3. It should build CSS and JS into the `build` folder
4. Open the site, it should load the CSS
5. Create a new post
6. Insert a columns block
7. It should have the "Column with Left Heading" option

Try the same with `yarn workspace wporg-parent-2021 start` — it should start a watch script that builds when the files change.
